recipes/stir/meta.yaml

```diff
@@ -0,0 +1,52 @@
+
+{% set name = "STIR" %}
+{% set version = "v3_0_1" %}
+{% set sha256 = "4f257fa7a4b15c9d6eddf981d80b79fc27a3ed10df4b8184d507d82a22f3596e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/olivierverdier/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [py2k]
+  number: 0
+
+requirements:
+  build:
+    - python
+    - pip
+    - cmake
+    - boost
+    - swig
+    - libgcc
+    - toolchain
+  run:
+    - python
+    - boost
+    - swig
+    - libgcc
+
+test:
+  imports:
+    - stir
+
+about:
+  home: http://stir.sourceforge.net/
+  license: GPL and LGPL
+  license_family: GPL
```

Comment:
Can you please include a `license_file`?

Reply:
Ok!